.vscode/settings.json

```diff
@@ -57,9 +58,11 @@
     "ruff.logLevel": "error",
     "extensions.ignoreRecommendations": false,
     "search.exclude": {
-      "tests/legacy": true
+      "tests/static": true,
+      "tests/unscheduled": true,
```

Comment:
Any reason to exclude unschedule? They're just regular Python tests, just haven't been SFI'd anywhere yet.